rundeckapp/grails-app/views/menu/home.next.gsp

```diff
@@ -104,7 +103,6 @@
   </div>
   <!-- VUE JS MODULES -->
   <asset:stylesheet href="static/css/pages/home.css"/>
-  <asset:javascript src="static/components/version-notification.js"/>
```

Comment:
This file isn't being created. In fact the components in the folder version-notification do not seem to be used anywhere.